Features/links.feature

```diff
@@ -0,0 +1,529 @@
+    En bevat de _links.self van de collectie deze parameters niet
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En een client geeft bepaalde parameters niet op (bv. page, pageSize, geldigOp, beschikbaarOp)
+    En de default waarde wordt gebruikt voor deze parameters
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van de collectie deze parameters niet
+
+  Scenario: self link van een resource in een collectie (niet zijnde een levenscyclus endpoint)
+    Als er een bevraging wordt gedaan op één van de endpoints
+    En deze endpoint levert een collectie van resources
+    En de geldigOp parameter wordt meegegeven
+    En de beschikbaarOp parameter wordt meegegeven
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link.self de meegegeven tijdreisparameter geldigOp 
+    En bevat de link.self de meegegeven tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En de geldigOp parameter wordt meegegeven
+    En de beschikbaarOp parameter wordt niet meegegeven
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link.self de meegegeven tijdreisparameter geldigOp 
+    En bevat de link.self niet de tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En de geldigOp parameter wordt niet meegegeven
+    En de beschikbaarOp parameter wordt meegegeven
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
+    En bevat de link.self niet de tijdreisparameter geldigOp 
+    En bevat de link.self de meegegeven tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En de geldigOp parameter wordt niet meegegeven
+    En de beschikbaarOp parameter wordt niet meegegeven
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID
```

Comment:
Als zowel de geldigOp als beschikbaarOp parameter niet meegegeven wordt, dan bevat de self link geen tijdreis url toch? Hoe noemen we dat endpoint dan wel eigenlijk, regulier resource endpoint?

Reply:
Dit zou idd wel gewoon tijdreis endpoint genoemd kunnen worden. Want als je geen parameters meegeeft dan wordt er impliciet toch getijdreisd, zei het met geldigOp=vandaag, beschikbaarOp=nu